.idea/runConfigurations.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why is this being deleted? I'm not saying its a bad thing but what's the justification as we need to be careful with deleting stuff, especially .xml set-ups.